quad/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "quad"
+description = "Quadrature integration"
+authors = [
+  "Alessandro Candido <candido.ale@gmail.com>",
+  "Alex Raiteri <alex.raiteri@studenti.unimi.it>",
+]
+readme = "../README.md"
+
+categories.workspace = true
+edition.workspace = true
+keywords.workspace = true
+license.workspace = true
+repository.workspace = true
+version.workspace = true
+
```

Comment:
```suggestion

[lib]
name = "quad"
crate-type = ["cdylib", "rlib"]
```

Reply:
This should be optional, isn't it?

The name is inferred from the package name.
https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-name-field

For crate type the default is `lib`.
https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-crate-type-field
That means it will infer on its own which library type is best suited.
https://doc.rust-lang.org/reference/linkage.html
For Python we explicitly need the `.so`, but in this crate should be fine with the default.

Reply:
That's right, I didn't know that.